website/content/docs/updates/important-changes.mdx

```diff
@@ -15,16 +15,30 @@ valid_change_types: >-
 
 # Important changes
 
-**Last updated**: 2025-06-05
+**Last updated**: 2025-06-12
 
 Always review important or breaking changes and remediation recommendations
 before upgrading Vault.
 
-## New behavior 
+## New behavior
 
-None.
+## Key Pair Authentication for Snowflake DB Secrets Engine
+
+| Change       | Affected version | Affected deployments
+| ------------ | ---------------- | --------------------
+| New behavior | 1.20.0           | any
+
+As of version 1.20.0, Vault has added support for [authenticating to Snowflake
+databases using key pairs](/vault/api-docs/secret/databases#configure-connection).
+Snowflake is disabling password authentication for all users in November 2025,
+and password authentication will eventually be removed in subsequent Vault releases.
```

Comment:
```suggestion
As of version 1.20.0, Vault supports
[Snowflake database authentication using key pairs](/vault/api-docs/secret/databases#configure-connection)
as an alternative to password authentication, which Snowflake plans to disable
in November 2025. Vault support for password authentication with Snowflake is
now deprecated and will be removed in a future release.
```

It sounds like we're effectively deprecating the associated functionality. We should also update the deprecations page with the deprecation notice date (JUN 2025) and expected EOL (even if it's something general like CYQ2 2026).

Reply:
Good call, @schavis! We made sure to include a deprecation notice earlier in the release cycle, so I can confirm that the deprecation notice has been posted to our docs!
https://developer.hashicorp.com/vault/docs/updates/deprecation#snowflake-db-password-authentication